package/boot/uboot-tools/Makefile

```diff
@@ -2,7 +2,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_DISTNAME:=u-boot
 PKG_VERSION:=2025.01
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
I don't think PKG_RELEASE bump is necessary as the resulting binaries should be identical.

Reply:
Yeah, I think you are right.